--- a/SConstruct
+++ b/SConstruct
@@ -715,7 +715,7 @@ else:
         confdefs.append("#define SUN_LEN(ptr) ((size_t) (((struct sockaddr_un *) 0)->sun_path) + strlen((ptr)->sun_path))\n")
         confdefs.append("#endif /* SUN_LEN */\n")
 
-    if config.CheckHeader(["bits/sockaddr.h", "linux/can.h"]):
+    if config.CheckHeader(["sys/socket.h", "linux/can.h"]):
         confdefs.append("#define HAVE_LINUX_CAN_H 1\n")
         announce("You have kernel CANbus available.")
     else:
