--- a/SConstruct
+++ b/SConstruct
@@ -774,7 +774,7 @@ else:
                         "+ strlen((ptr)->sun_path))\n")
         confdefs.append("#endif /* SUN_LEN */\n")
 
-    if config.CheckHeader(["bits/sockaddr.h", "linux/can.h"]):
+    if config.CheckHeader(["sys/socket.h", "linux/can.h"]):
         confdefs.append("#define HAVE_LINUX_CAN_H 1\n")
         announce("You have kernel CANbus available.")
     else:
