--- a/pylib/wscript
+++ b/pylib/wscript
@@ -1,5 +1,4 @@
 from waflib import Utils  # pylint: disable=import-error
-from wafhelpers.fix_python_config import FixConfig
 
 
 def options(opt):
@@ -7,9 +6,8 @@ def options(opt):
 
 
 def configure(conf):
-    fixed = FixConfig(conf)
-    fixed.load('python')
-    fixed.check_python_version((2, 6, 0))
+    conf.load('python')
+    conf.check_python_version((2, 6, 0))
     conf.check_python_headers(features='pyext')  # Extension-only, no embedded
 
 
