--- a/wafhelpers/configure.py
+++ b/wafhelpers/configure.py
@@ -61,9 +61,6 @@ def cmd_configure(ctx, config):
     for opt in opt_map:
         ctx.env[opt] = opt_map[opt]
 
-    from wafhelpers.check_compiler import check_compiler
-    check_compiler(ctx)
-
     if ctx.options.enable_rtems_trace:
         ctx.find_program("rtems-tld", var="BIN_RTEMS_TLD",
                          path_list=[ctx.options.rtems_trace_path,
