--- a/ntpq/wscript
+++ b/ntpq/wscript
@@ -24,7 +24,7 @@ def build(ctx):
 		target		= "ntpq",
 		features	= "c cprogram bld_include src_include libisc_include ntp_version",
 		source		= ntpq_source,
-		use			= "ntp opts isc M RT READLINE EDIT PTHREAD CRYPTO CURSES READLINE",
+		use			= "ntp opts isc M RT READLINE EDIT PTHREAD CRYPTO CURSES NCURSES READLINE",
 		includes	= [
 						"%s/ntpq/" % bldnode
 		],
--- a/wafhelpers/configure.py
+++ b/wafhelpers/configure.py
@@ -269,7 +269,9 @@ def cmd_configure(ctx, config):
 	ctx.check_cc(lib="m", comment="Math library")
 	ctx.check_cc(lib="rt", mandatory=False, comment="realtime library")
 	ctx.check_cc(lib="curses", mandatory=False, comment="curses library, required for readline on OpenBSD") # Required for readline on OpenBSD.
+	ctx.check_cc(lib="ncurses", mandatory=False, comment="ncurses library, required for readline on LEDE") # Required for readline on LEDE.
 	ctx.check_cc(lib="readline", use="CURSES", mandatory=False, comment="readline library")
+	ctx.check_cc(lib="readline", use="NCURSES", mandatory=False, comment="readline library")
 
 	# Find OpenSSL. Must happen before function checks
 	if ctx.options.enable_crypto:
