--- a/libntp/machines.c
+++ b/libntp/machines.c
@@ -85,7 +85,7 @@ int ntp_gettime(struct ntptimeval *ntv)
 
 	ZERO(tntx);
 	/* not intercepted because ntpd doesn't use this */
-	result = ntp_adjtime(&tntx);
+	result = adjtimex(&tntx);
 	ntv->time = tntx.time;
 	ntv->maxerror = tntx.maxerror;
 	ntv->esterror = tntx.esterror;
--- a/ntpd/ntp_intercept.c
+++ b/ntpd/ntp_intercept.c
@@ -599,7 +599,7 @@ int intercept_ntp_adjtime(struct timex *
 	    exit(1);
 	}
     } else {
-	res = ntp_adjtime(tx);
+	res = adjtimex(tx);
 
 	if (mode == capture)
 	    printf("ntp_adjtime " ADJFMT " %d\n",
