--- a/ntpd/ntpd.c
+++ b/ntpd/ntpd.c
@@ -485,7 +485,7 @@ const char *ntpd_version(void)
 {
     static char versionbuf[64];
     snprintf(versionbuf, sizeof(versionbuf),
-	     "ntpd ntpsec-%s+%d %s", VERSION, VCS_TICK, VCS_DATE);
+	     "ntpd ntpsec-%s", VERSION);
     return versionbuf;
 }
 
--- a/pylib/util.py
+++ b/pylib/util.py
@@ -79,9 +79,7 @@ def safeargcast(arg, castfunc, errtext,
 
 
 def stdversion():
-    return "ntpsec-%s+%s %s" % (ntp.version.VERSION,
-                                ntp.version.VCS_TICK,
-                                ntp.version.VCS_DATE)
+    return "ntpsec-%s" % (ntp.version.VERSION)
 
 
 def rfc3339(t):
