--- a/wafhelpers/check_compiler.py
+++ b/wafhelpers/check_compiler.py
@@ -31,7 +31,7 @@ int main(void) {
 
 def check_compiler(ctx):
 
-    if ctx.env.ENABLE_CROSS:
+    if ctx.options.cross_compiler:
         return
 
     defines = {
--- a/wafhelpers/check_openssl.py
+++ b/wafhelpers/check_openssl.py
@@ -32,10 +32,14 @@ def configure_openssl(ctx):
         OPENSSL_LIB = False
 
     if OPENSSL_HEADERS and OPENSSL_LIB:
+        execute = True
+        if ctx.env.ENABLE_CROSS:
+            execute = False
+
         ctx.check_cc(
             fragment=OPENSSL_FRAG % "\n".join(["#include <%s>" % x
                                                for x in headers]),
-            execute=True,
+            execute=execute,
             use="CRYPTO",
             msg="Checking if OpenSSL works",
             comment="OpenSSL support"
