--- a/ntpd/ntp_sandbox.c
+++ b/ntpd/ntp_sandbox.c
@@ -420,13 +420,24 @@ int scmp_sc[] = {
 	/* gentoo 64-bit and 32-bit, Intel and Arm use mmap */
 	SCMP_SYS(mmap),
 #endif
-#if defined(__i386__) || defined(__arm__)
-	SCMP_SYS(_newselect),
+#if defined(__i386__) || defined(__arm__) || defined(__mips__)
 	SCMP_SYS(_llseek),
 	SCMP_SYS(mmap2),
-	SCMP_SYS(send),
 	SCMP_SYS(stat64),
 #endif
+#if defined(__i386__) || defined(__arm__)
+	SCMP_SYS(_newselect),
+	SCMP_SYS(send),
+#endif
+#ifdef __mips__
+	SCMP_SYS(getegid),
+	SCMP_SYS(getgid),
+	SCMP_SYS(getuid),
+	SCMP_SYS(ipc),
+	SCMP_SYS(listen),
+	SCMP_SYS(readv),
+	SCMP_SYS(writev),
+#endif
 };
 	{
 		for (unsigned int i = 0; i < COUNTOF(scmp_sc); i++) {
